doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
-
-<p align="center"><img src="./figures/tech_scope_3.png" alt="scope" title="Document Types" width="100%"/></p>
-<p align="center"><b>Figure 2:</b> Description of the possible different levels of CNTT artefacts</p>
-
-<a name="3.0"></a>
-## 3. Principles
-
-Any specification work created within CNTT **must** conform to set of principles specified by CNTT:
-
-* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
-* [Reference Architecture Principles](../ref_arch#principles)
-
-<a name="4.0"></a>
-## 4. Relevant Technologies
-This section describes the relevant technologies for CNTT and clarifies CNTT position about them. The list below is not exhaustive but it is representative of existing trends. Chapter 5 describe CNTT policies of managing the Non-Conferment ones. 
-
-<a name="4.1"></a>
-### 4.1 IO Virtualization
-
-There are different ways of which IO devices (such as NICs) are presented to workloads for consumption by those workloads. Here is a list of current methods of existing IO Virtualization:
-
-- Para-Virtualization method (software only).
-- Direct Assignment via h/w assisted PCI-Passthrough (IOMMU).
-- Device Sharing with SR-IOV & h/w assisted PCI-Passthrough (IOMMU).
-- Para-Virtualization method with Hardware support.
+<a name="1.1"></a>
+## 1.1 Glossary
+The definition and intent of the terminology used throughout the documents is defined in the [Glossary](./glossary.md).
 
-**Figure 3** below shows some of the relevant IO Virtualization techniques.
+<a name="1.2"></a>
+## 1.2 Problem Statement
+Based on informal conversations with many operators and developers, there is a realization that there are significant technical, operational and business challenges to the development and deployment of VNF applications related to the lack of a common virtualized infrastructure platform.  These include but are not limited to the following:
 
-<p align="center"><img src="./figures/tech_iov.png" alt="scope" title="Document Types" width="70%"/></p>
-<p align="center"><b>Figure 3:</b> Relevant IO Virtualization Techniques</p>
+ - Higher development costs due to the need to develop Virtual Network Functions (VNF) on multiple custom platforms for each operator
+ - Increased complexities due to the need to maintain multiple versions of applications to support each custom environemnt
+ - Lack of Testing and validation commonalities, leading to inefficiencies and increased time to market. While the operators will still do internal testing, but using an industry driven verification program based on a common NFVI would provide a head start.
+ - Slower adoption of cloud-native NFV applications and architectures.  A Common Telco NFVI may provide an easier path to methodologies that will drive faster cloud-native NFV application development.
```

Comment:
Given the discussion in #1168 can I suggest changing `A Common Telco NFVI` to `A Common Telco Cloud Infrastructure`?

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/e95099e73194fe029a164333553449662c1f1a61